devel/libwebsockets/Portfile

```diff
@@ -39,7 +41,7 @@ configure.args-append \
                     -DLWS_WITH_LIBUV=ON \
                     -DLWS_WITH_RANGES=ON \
                     -DLWS_WITH_SOCKS5=ON \
-                    -DLWS_WITH_ZIP_FOPS=ON
+                    -DLWS_WITH_ZIP_FOPS=ON \
```

Comment:
Trailing slash should not be added here as there is no line after it.

Reply:
done